main/src/ca/uwaterloo/flix/language/phase/jvm/GenHoleErrorClass.scala

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2021 Jonathan Lindegaard Starup
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ca.uwaterloo.flix.language.phase.jvm
+
+import ca.uwaterloo.flix.api.Flix
+import org.objectweb.asm.{ClassWriter, Label}
+import org.objectweb.asm.Opcodes._
+
+object GenHoleErrorClass {
+
+  val holeFieldName: String = "hole"
```

Comment:
Is the string in the HoleError used in meaningful way? When I run code with `???` it gives the message `Hole '?h22649' at examples\misc\test.flix:3:5` is this meaningful or could it just say `Hole error at examples\misc\test.flix:3:5`?

Reply:
I would say so- its the name of the symbol.

Reply:
E.g. ?myHoleName

Reply:
I see! wasn't aware of that feature